01_IOT/boards/nrf7002dk_nrf5340_cpuapp.conf

```diff
@@ -16,10 +16,10 @@ CONFIG_WPA_SUPP=y
 
 # These should be in a board file for the nRF7002:
 # (Choose only one that applies or none for no WiFi password)
+# CONFIG_STA_KEY_MGMT_NONE=y
 CONFIG_STA_KEY_MGMT_WPA2=y
```

Comment:
@szczys is there a reason why these are in the board `.conf` vs. the app's `prj.conf`? Seems like Wi-Fi security, SSID, etc should be configurable at the project level vs. being stored with the board?

Reply:
Sorry, disregard previous comment, I got mixed up. I forgot these apps are intended to support different boards with WiFi, cellular, etc, and these `<board>.conf` files are the _app-specific_ board config files, rather than the board conf in the board definitions.